connector/connect/client/jvm/src/test/scala/org/apache/spark/sql/PlanGenerationTestSuite.scala

```diff
@@ -1964,6 +1964,34 @@ class PlanGenerationTestSuite
     fn.to_csv(fn.col("d"), Collections.singletonMap("sep", "|"))
   }
 
+  functionTest("str_to_map") {
```

Comment:
please also add test for 
`def str_to_map(text: Column, pairDelim: Column)`
`def str_to_map(text: Column): Column`